src/inspector_agent.cc

```diff
@@ -72,15 +72,23 @@ std::string StringViewToUtf8(const StringView& view) {
   }
 
   // CHAKRA-TODO: Figure out what to do here
-  assert(false);
-
   const uint16_t* source = view.characters16();
+  const size_t length = view.length();
+  std::string result;
+
+  for (size_t i = 0; i < length; i++) {
+    uint16_t wCh = source[i];
+    assert((wCh & 0xFF80) == 0);
+
+    result.append(1, static_cast<char>(wCh));
+  }
+
```

Comment:
This is obviously a hack and will assert if the assumption is broken.  Once we have ICU support it should be trivial to fix, but even if we don't we'll have to revisit.